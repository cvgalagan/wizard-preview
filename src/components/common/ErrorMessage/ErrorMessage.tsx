import React from "react"
import styles from "./ErrorMessage.module.scss"

interface Props {
    text: string
}

const ErrorMessage: React.FC<Props> = props => {
    return <div className={styles.errorMessage}>{props.text}</div>
}

export default ErrorMessage
