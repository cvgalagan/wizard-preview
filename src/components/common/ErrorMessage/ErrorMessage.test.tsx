import React from "react"
import { shallow } from "enzyme"
import ErrorMessage from "./ErrorMessage"

describe("ErrorMessage", () => {
    it("should render", () => {
        const wrapper = shallow(<ErrorMessage text="test" />)
        expect(wrapper).toMatchSnapshot()
    })
})
