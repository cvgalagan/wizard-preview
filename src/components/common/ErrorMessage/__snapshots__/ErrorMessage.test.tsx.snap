// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render 1`] = `
<div
  className="errorMessage"
>
  test
</div>
`;
